README.md

```diff
@@ -1 +1,105 @@
-# portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="https://raw.githubusercontent.com/microverseinc/readme-template/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  
+</div>
+
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+ - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- - [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+# 📖 <a name="about-project">Portfolio</a>
+> In this project, I create my portfolio, I can call it, my cv, certainly, it's evolved day by day, but in December of this year(2023), it will be pretty and presentable with the help of Microverse.
+## 🛠 Built With <a name="built-with">HTML, CSS and Linter</a>
+### Tech Stack <a name="tech-stack"></a>
+> In this project we use just one page html and his document css to make it pretty
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">index2.html</a></li>
+    <li><a href="#">styles2.css</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+### Key Features <a name="key-features">Portfolio</a>
+> In this project, I just introduced myself and I give my contacts
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+> 
+- [Live Demo Link](Coming soon...)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+## 💻 Getting Started <a name="getting-started"></a>
+> To do the same as this project, use the template as pattern, after build the same as you in the template.
+Use the template for the portfolio, you will find it at Fring
+
+### Setup
+Clone this repository to your desired folder:
+git clone https://github.com/francksefu/portfolio.git
+
+### Usage
+To run the project, open the html folder in your navigator
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- AUTHORS -->
+## 👥 Authors <a name="authors">Franck Sefu Kalunga</a>
+
+👤 Franck Sefu Kalunga
+- GitHub: [@githubhandle](https://github.com/francksefu)
+- Twitter: [@twitterhandle](https://twitter.com/franck_sefu)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/franck-sefu-884705254/)
+👤 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FUTURE FEATURES -->
+## 🔭 Future Features <a name="future-features">Make it dynamic</a>
+> Describe 1 - 3 features you will add to the project.
```

Comment:
- [ ] I would suggest you removing this kind of lines as these are meant for the writer and not for the visitor of this readme.